ChatSecure/Classes/Model/Yap Storage/OTRXMPPBuddy.h

```diff
@@ -11,6 +11,8 @@
 
 @class XMPPvCardTemp;
 
+extern NSString * _Nonnull const OTRBuddyPendingApprovalDidChangeNotification;
```

Comment:
Add documentation string describing userInfo buddy key.

`/** Contains userInfo with buddy object in "buddy" key */`